src/components/layouts/KahootLayout.tsx

```diff
@@ -18,10 +17,9 @@ interface KahootLayoutProps {
   children: React.ReactNode;
 }
 function KahootLayout({ children }: KahootLayoutProps) {
-  const dispatch = useDispatch<AppDispatch>();
   const history = useHistory();
   const handleLogout = () => {
-    dispatch(logout());
+    store.dispatch(logout());
```

Comment:
You shouldn't use store.dispatch let use const dispatch = useDispatch<AppDispatch>();